sormas-ui/src/main/java/de/symeda/sormas/ui/labmessage/LabMessageController.java

```diff
@@ -761,4 +802,36 @@ private void addProcessedInMeantimeCheck(CommitDiscardWrapperComponent createCom
 			}
 		});
 	}
+
+	private HorizontalLayout getLabMessageButtonsPanel(LabMessageDto labMessage, Runnable callback) {
+		HorizontalLayout buttonsPanel = new HorizontalLayout();
+		buttonsPanel.setMargin(false);
+		buttonsPanel.setSpacing(true);
+		buttonsPanel.setWidth(100, Sizeable.Unit.PERCENTAGE);
+
+		Button deleteButton = ButtonHelper.createButtonWithCaption(Captions.actionDelete, I18nProperties.getCaption(Captions.actionDelete), (e) -> {
+			VaadinUiUtil.showDeleteConfirmationWindow(
+				String.format(I18nProperties.getString(Strings.confirmationDeleteEntity), I18nProperties.getCaption(Captions.LabMessage)),
+				() -> {
+					FacadeProvider.getLabMessageFacade().deleteLabMessage(labMessage.getUuid());
```

Comment:
It would be good to check whether a labMessage has been processed in the meantime before deleting.

Reply:
There is something like that [here](https://github.com/hzi-braunschweig/SORMAS-Project/blob/bd31430d5875770be7bf7b1e37386533ae0e5836/sormas-ui/src/main/java/de/symeda/sormas/ui/labmessage/LabMessageController.java#L755).